api/src/main/kotlin/edu/wgu/osmt/richskill/RichSkillEsRepo.kt

```diff
@@ -67,30 +79,39 @@ class CustomRichSkillQueriesImpl @Autowired constructor(override val elasticSear
             // boolQuery.should for logical OR
             skillName.nullIfEmpty()?.let {
                 if (it.contains("\"")) {
-                    bq.must(simpleQueryStringQuery(it).field("${RichSkillDoc::name.name}.raw").defaultOperator(Operator.AND))
+                    bq.must(
```

Comment:
Can we please revert the broad whitespace changes not related to the code we a re changing. We call this "boyscouting" (leaving it better than we found it). There a place for this, but not in a feature branch or during feature development.

Reply:
gotcha